### PR TITLE
Tune homepage texts

### PR DESCRIPTION
**Pull request by gorarakelyan**
- Tuned homepage texts
- Updated colab example link
- Updated demo link

### CODE REVIEW

**Review by roubkar — APPROVED** [review]

File: aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx [hunk 1]
```diff
@@ -15,37 +15,46 @@ export interface IExploreCard {
...
+    path: 'params',
+    icon: 'params',
   },
   {
-    title: 'Bookmarks',
+    title: 'Scatters Explorer',
     description:
-      "Aim UI uses segment's analytics toolkit to collect basic info about...",
-    path: 'bookmarks',
+      'Scatter explorer helps to explore and learn relationship, correlations, and clustering effects between metrics and parameters.',
```
Comment (SGevorg) [inline aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx:52]: Scatters Explorer or Scatter Explorer?

**Review by SGevorg — CHANGES_REQUESTED** [review]

File: aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx (hunk 1 above)
Comment (alberttorosyan) [reply aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx:52]: maybe Scatter Plot Explorer?

Comment (gorarakelyan) [reply aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx]: @alberttorosyan too long I think

**Comment by gorarakelyan** [comment]
@SGevorg is there anything else that needs to be changed?

File: aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx [hunk 2]
```diff
@@ -15,37 +15,46 @@ export interface IExploreCard {
...
+    description:
+      'View all your runs holistically on Runs Explorer: all hyperparameters, all metric last values',
     path: 'runs',
+    icon: 'runs',
   },
   {
     title: 'Metrics Explorer',
     description:
-      "Aim UI uses segment's analytics toolkit to collect basic info about...",
+      'Compare 100s of metrics in a few clicks with Metrics Explorer',
```
Comment (SGevorg) [inline aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx:31]: on Metrics Explorer

File: aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx [hunk 3]
```diff
@@ -15,37 +15,46 @@ export interface IExploreCard {
...
     path: 'metrics',
+    icon: 'metrics',
   },
   {
-    title: 'Params Explorer',
+    title: 'Images Explorer',
     description:
-      "Aim UI uses segment's analytics toolkit to collect basic info about...",
-    path: 'params',
+      'Track intermediate images and search, compare them on the Images Explorer',
```
Comment (SGevorg) [inline aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx:38]: on Images Explorer

File: aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx [hunk 4]
```diff
@@ -15,37 +15,46 @@ export interface IExploreCard {
...
+    path: 'images',
+    icon: 'images',
   },
   {
-    title: 'Tags',
-    description: 'We constantly seek to improve Aim for the community.',
-    path: 'tags',
+    title: 'Params Explorer',
+    description:
+      'Params explorer enables a parallel coordinates view for metrics and params',
```
Comment (SGevorg) [inline aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx:45]: The Params Explorer

File: aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx [hunk 5]
```diff
@@ -15,37 +15,46 @@ export interface IExploreCard {
...
+    path: 'params',
+    icon: 'params',
   },
   {
-    title: 'Bookmarks',
+    title: 'Scatters Explorer',
     description:
-      "Aim UI uses segment's analytics toolkit to collect basic info about...",
-    path: 'bookmarks',
+      'Explore and learn relationship, correlations, and clustering effects between metrics and parameters with Scatters Explorer',
```
Comment (SGevorg) [inline aim/web/ui/src/pages/Home/components/ExploreAim/ExploreAim.tsx:52]: on Scatters Explorer

**Review by SGevorg — CHANGES_REQUESTED** [review]

**Pull request merged**